### ISSUE TITLE
Variables in child custom components break

### ISSUE THREAD

**Issue by kobyl** [bencompton/framework7-react#48]
JS Fiddle: https://jsfiddle.net/2hu61z9h/3/

You'll see: Uncaught Error: Objects are not valid as a React child (found: object with keys {tag})... in the console.

The problem is let x = 10 and then embedding x into the child component. Remove that an all is well.

**Comment by renatoholanda** [comment]
Try putting .toString() to explicit output.

**Comment by kobyl** [comment]
That works in the child component, but that means I need to design my child components in a certain way, which kinda sucks.

I've been trying to dig into this though - where does the framework treat this.props.children? Because things are generated it's a little hard to follow.

**Comment by bencompton** [comment]
Thanks for the issue report and the fiddle! Framework7 React appears to have a lacking in the way it processes children where it does not handle this particular scenario.
